src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1693,6 +1660,41 @@
       "required": [ "tool" ]
     },
 
+
+    "runAutomationDetails": {
+      "description": "Information that describes a run's identity and role within an engineering system process.",
+      "additionalProperties": false,
+      "type": "object",
+      "properties": {
+
+        "description": {
+          "description": "A description of the role played within the engineering system by this object's containing run object.",
+          "$ref": "#/definitions/message"
+        },
+
+        "instanceId": {
+          "description": "A hierarchical string that uniquely identifies this object's containing run object.",
+          "type": "string"
+        },
+
+        "instanceGuid": {
+          "description": "A string that uniquely identifies this object's containing run object in the form of a GUID.",
```

Comment:
>string that uniquely identifies [](start = 28, length = 31)

The spec might not be uniform in its phrasing. I like the formulation "A stable, unique identifier for <the thing> in the form of a GUID."